### DISCUSSION TITLE
Error running Gradle (key store issue)

### DISCUSSION THREAD

**Discussion by dylanirlbeck** [subosito/flutter-action#129] [General]
Hi all. So I'm relatively new to Flutter, but I currently have this Action running inside a repo to check that the app can build properly. It was passing for a while, but ever since I added a [key store](https://flutter.dev/docs/deployment/android#create-a-keystore) to the app (to prep for an Android release), `flutter-action` has been failing to build in CI with the following error:

```
ProcessException: Process "/home/runner/work/..." exited abnormally:
Starting a Gradle Daemon (subsequent builds will be faster)
  Command: /home/runner/work/.../android/gradlew app:properties
```

My question is, what is the recommended way to deal with the key store file, since you're not supposed to check it in to source control? Thanks in advance!

**Comment by emme1009** [comment]
@dylanirlbeck  did you find a solution for this ?

**Comment by dylanirlbeck** [comment]
I didn't no :/ We now just live with the failing build.

**Comment by emme1009** [comment]
@dylanirlbeck I found a solution. Just add the keys as secrets in github and pass them along as "env" on the build command. Error is gone and app is signed correctly.

**Comment by guidezpl** [comment]
Hey, I just did this setup for https://github.com/flutter/gallery, you can recreate the keystore using environment secrets: https://github.com/flutter/gallery/pull/488/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R121
